Dockerfile

```diff
@@ -0,0 +1,73 @@
+
+# Adds DeepX Gazebo Sensors image.
+#
+# Docker commands:
+# - To build the image:
+#   docker build -t deepx-gz-sensors .
+# - To publish the image:
+#   docker push ghcr.io/deepx-inc/deepx-gz-sensors
+
+FROM ghcr.io/deepx-inc/base_images:foxy
+
+# ---BUILD TOOLS---
+RUN apt -qq update \
+    && apt -qq install -y --no-install-recommends \
+    cmake \
+    ffmpeg \
+    freeglut3-dev \
+    g++-8 \
+    git \
+    gnupg \
+    libfreeimage-dev \
+    libglew-dev \
+    libogre-2.2-dev \
+    libxi-dev \
+    libxmu-dev \
+    ninja-build \
+    pkg-config \
+    redis-server \
+    redis-tools \
+    software-properties-common \
+    lsb-release \
+    wget \
+    curl
+
+# ---STANDARD GAZEBO INSTALL---
+RUN apt -qq update && \
+    sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list' && \
+    sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-prerelease `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-prerelease.list' && \
+    wget http://packages.osrfoundation.org/gazebo.key -O - | apt-key add - && \
+    add-apt-repository ppa:kisak/kisak-mesa && \
+    curl -sSL http://get.gazebosim.org | sh && \
+    update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-8 800 --slave /usr/bin/g++ g++ /usr/bin/g++-8 --slave /usr/bin/gcov gcov /usr/bin/gcov-8 && \
+    apt -qq update && \
+    apt -qq upgrade -y --no-install-recommends \
+    libignition-cmake2-dev \
+    libignition-common4-dev \
+    libignition-fuel-tools7-dev \
+    libignition-gazebo6-dev \
+    libignition-gui6-dev \
+    libignition-launch5-dev \
+    libignition-math6-dev \
+    libignition-msgs8-dev \
+    libignition-physics5-dev \
+    libignition-plugin-dev \
+    libignition-tools-dev \
+    libignition-transport11-dev \
+    libignition-utils-dev \
+    libsdformat12 \
+    libignition-plugin-dev \
+    && apt -qq -y autoclean \
+    && apt -qq -y autoremove \
+    && rm -rf /var/lib/apt/lists/*
+
+WORKDIR "/root/gazebo"
+
+# ---PREPARE GZ-RENDERING DIRECTORY---
+RUN git clone -b 2022.10.21 --single-branch https://github.com/DeepX-inc/gz-rendering.git && \
```

Comment:
The gazebo dockerfile clones the latest version of the main branch. We should either do the same here, or change the gazebo dockerfile to pull the same tag.
The latter option is technically more correct, but on the other hand we don't use fixed versions for the apt gazebo libraries either.

Reply:
Sorry which gazebo Dockerfile are you talking about?

Reply:
Here: https://github.com/DeepX-inc/deepx_docker_files/blob/main/base_images/gazebo/Dockerfile#L97

Reply:
Ah OK. And by `we don't use fixed versions for the apt gazebo libraries either.` you mean the one in [here](https://github.com/DeepX-inc/deepx_docker_files/blob/7f26f6119dbb01b6d1f1c34ac7ecda2313774faa/base_images/gazebo/Dockerfile#L56)?

Reply:
In that case maybe it will take more time to fix it. Perhaps for this sprint I can change it to the former first. Further improvement can be done as part of the other Dockerfile optimization task.

Reply:
Right. We _can_ use fixed versions for the `gz-rendering` and `gz-sensors` libraries, but in either case we should be consistent in the approach.

Reply:
Change it to the former for now.